### ISSUE TITLE
A11y_VisualStudioCodeClient_Customizations and settings_Keyboard shortcuts_Screenreader:NVDA/Narrator is announcing Keybendings results twice

### ISSUE THREAD

**Issue by VijayaBharathyA** [microsoft/vscode#159296]
### GitHub Tags:
#A11yMAS; #A11yTCS; #A11ySev3; #DesktopApp; #Win32; #Visual Studio Code Client; #BM-VisualStudioCodeClient-Win32-Aug2022; #WCAG1.3.1; #NVDA; #ScreenReader; #Narrator; 

### Environment and OS details:
Visual Studio Code Version: 1.70.2 (user setup)
Windows 11 Pro version 21H2 (OS Build 22000.856)

### Repro Steps:

1. Turn on NVDA
2. Open Visual studio code editor.
3. Navigate to setting icon
4. Select "Keyboard shortcuts options"
5. Seach name in search bar
6. Verify that screen reader announcing the results of keyboard shortcuts multiple twice or not.

### Actual Result:
NVDA/Narrator is announcing Keybendings results twice. 

### Expected Result:

NVDA/Narrator should not announce Keybendings results twice. 

### Attachment:

![A11y_VisualStudioCodeClient_Customizations and settings_Keyboard shortcuts_NVDA](https://user-images.githubusercontent.com/100189407/186893090-e37dcd4d-f2bb-4264-aa62-4d970ef667cb.jpg)

![A11y_VisualStudioCodeClient_Customizations and settings_Keyboard shortcuts_Narrator](https://user-images.githubusercontent.com/100189407/186893084-28f4c9f8-44fa-40fc-850a-2ac50fcc2726.jpg)

**Comment by isidorn** [comment]
@VijayaBharathyA thanks for reporting this.
@sandy081 I can reproduce this. It seems that for every query the aria label of the list is read twice. Might it be that we focus the list two times? Which causes the screen reader to re-read.
Or can we de-bounce the refresh of the list by 200ms?

This is a sev3 issue, which means we can close it if we do not want to work on it. But overall I think the issue makes sense.

**Comment by sandy081** [comment]
We are not focusing the list twice but I suspect aria live element aria label is getting updated with the same when the filter changes. When I try to repro locally (on Mac) I see that this aria live element is getting updated only once with the sample filter **auto**.

**Comment by isidorn** [comment]
I could only repro on Windows.
As I mentioned feel free to close as not planned, if there is nothing obvious to fix here.
